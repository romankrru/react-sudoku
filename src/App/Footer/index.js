import React from "react";
import styles from "./assets/index.module.css";

const Footer = () => (
	<footer className={styles.footer}>
		<a
			href="https://github.com/romankrru/react-sudoku"
			className={styles.footerLink}
		>
			View Source on Github
		</a>
	</footer>
);

export default Footer;
