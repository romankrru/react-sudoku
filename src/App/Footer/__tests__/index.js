import React from "react";
import { render, screen } from "@testing-library/react";

import Footer from "../";

it("<Footer /> renders without crashes", () => {
	const view = render(<Footer />);

	expect(screen.getByText("View Source on Github")).toBeInTheDocument();
	expect(view.container).toMatchSnapshot();
});
