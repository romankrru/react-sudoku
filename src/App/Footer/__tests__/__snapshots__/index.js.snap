// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders without crashes 1`] = `
<div>
  <footer
    class="footer"
  >
    <a
      class="footerLink"
      href="https://github.com/romankrru/react-sudoku"
    >
      View Source on Github
    </a>
  </footer>
</div>
`;
